lib/streamlit/elements/media.py

```diff
@@ -567,7 +567,7 @@ def marshall_video(
     if autoplay:
         ctx = get_script_run_ctx()
         proto.autoplay = autoplay
-        id = compute_widget_id(
+        id = compute_element_id(
```

Comment:
nit: we should perhaps use the opportunity of this PR and rename the variable `id` to `element_id` everywhere since the code is anyways already touched. The reason is that the linter (at least my local one) complains that `id` is a pre-defined Python keyword and we overshadow that by giving the variable the same name

Reply:
Renamed it everywhere 👍